### ISSUE TITLE
краткое описание товара

### ISSUE THREAD

**Issue by mozhayev** [yumashev/wooms#126]
Добрый день! на вордпрессе есть краткое описание товара, не плохо было бы реализовать возможность копировать с моего склада. (в карточки товара сервисом мой Склад ПРЕДУСМОТРЕНА возможность создать  Дополнительные поля) эта возможность бы повысила спрос на продукт

**Comment by artikus11** [comment]
Экспорт доп полей уже давно есть в платной версии. В настройках достаточно включить и все будет работать

**Comment by mozhayev** [comment]
Там сделано в раздел детали товара!
А я имею виду в краткое описания Поле без дела простаивает!
Да и там добавляться все поля кроме (типа текст) 



Отправлено из мобильной Почты Mail.Ru


суббота, 26 января 2019 г., 20:45 +0500 от notifications@github.com  <notifications@github.com>:

**Comment by artikus11** [comment]
На всех не угодить, не у всех краткое описание простаивает. Кто-то его использует для записи основного описания, кто-о вообще отключает. Это уже сугубо индивидуально надо решать, под каждый конкретный случай

**Comment by mozhayev** [comment]
Есть предложения ?



Отправлено из мобильной Почты Mail.Ru


суббота, 26 января 2019 г., 21:15 +0500 от notifications@github.com  <notifications@github.com>:

**Comment by uptimizt** [comment]
Пока была идея сделать опцию чтобы описание из МойСклад добавлялось в краткое описание вместо полного. Для тех у кого полное описание сложное и делается на сайте. Отсутствует на Складе.

Как доп поле - надо подумать.

Есть пример карточки товара в МойСклад где это поле добавлено и заполнено?

@mozhayev

**Comment by mozhayev** [comment]
Идея с переносом с полного на краткое описания отличная! оно важней так как на глазах! а полное описание редко кто смотрит
Но оно тоже бы не помешало 😁



Отправлено из мобильной Почты Mail.Ru


понедельник, 28 января 2019 г., 1:33 +0500 от notifications@github.com  <notifications@github.com>:

**Comment by Antonio2k** [comment]
Приветствую, случайно не решился вопрос о переносе описания из Мой Склад в краткое описание на WooCommerce?

**Comment by mozhayev** [comment]
Здравствуйте! нет вопрос не решён!



Отправлено из мобильной Почты Mail.ru


воскресенье, 17 ноября 2019 г., 0:03 +0500 от notifications@github.com  <notifications@github.com>:

**Comment by uptimizt** [comment]
дубль https://github.com/wpcraft-ru/wooms/issues/96

**Comment by mozhayev** [comment]
Здравствуйте возникает вот такая ошибка
Uncaught Error: Class 'WooMS\Abstract Walker' not found in /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/wooms-extra/inc/ProductGrouped.php:10
я так понял , что в фале
 /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/wooms-extra/inc/ProductGrouped.php отсутствует класс WooMS\AbstractWalker
после перехода на новый API

You are receiving this because you were mentioned.
Reply to this email directly,  view it on GitHub , or  unsubscribe .
 
 
--
Александр Можаев

**Comment by mozhayev** [comment]
[26-Jun-2020 12:02:03 UTC] PHP Fatal error:  Uncaught Error: Call to a member function is_virtual() on bool in /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/wooms-extra/inc/OrderSender.php:519
Stack trace:
#0 /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/wooms-extra/inc/OrderSender.php(463): WooMS\OrderSender::get_product_type(Object(WC_Order_Item_Product))
#1 /home/m/mozhayev/rybolov59.rf/public_html/wp-includes/class-wp-hook.php(287): WooMS\OrderSender::add_positions(Array, 272799)
#2 /home/m/mozhayev/rybolov59.rf/public_html/wp-includes/plugin.php(206): WP_Hook->apply_filters(Array, Array)
#3 /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/wooms-extra/inc/OrderSender.php(189): apply_filters('wooms_order_dat...', Array, 272799)
#4 /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/wooms-extra/inc/OrderSender.php(301): WooMS\OrderSender::update_order(272799)
#5 /home/m/mozhayev/rybolov59.rf/public_html/wp-includes/class-wp-hook.php(287): WooMS\OrderSender::batch_hadler()
#6 /home/m/mozhayev/rybolov59.rf/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters('', Array)
#7 /home/m/mozhayev/rybolov59.rf/public_html/wp-includes/plugin.php(544): WP_Hook->do_action(Array)
#8 /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/woocommerce/packages/action-scheduler/classes/actions/ActionScheduler_Action.php(22): do_action_ref_array('wooms_schedule_...', Array)
#9 /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/woocommerce/packages/action-scheduler/classes/abstracts/ActionScheduler_Abstract_QueueRunner.php(65): ActionScheduler_Action->execute()
#10 /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/woocommerce/packages/action-scheduler/classes/ActionScheduler_QueueRunner.php(162): ActionScheduler_Abstract_QueueRunner->process_action(652017, 'Async Request')
#11 /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/woocommerce/packages/action-scheduler/classes/ActionScheduler_QueueRunner.php(132): ActionScheduler_QueueRunner->do_batch(25, 'Async Request')
#12 /home/m/mozhayev/rybolov59.rf/public_html/wp-includes/class-wp-hook.php(287): ActionScheduler_QueueRunner->run('Async Request')
#13 /home/m/mozhayev/rybolov59.rf/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters('', Array)
#14 /home/m/mozhayev/rybolov59.rf/public_html/wp-includes/plugin.php(478): WP_Hook->do_action(Array)
#15 /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/woocommerce/packages/action-scheduler/classes/ActionScheduler_AsyncRequest_QueueRunner.php(52): do_action('action_schedule...', 'Async Request')
#16 /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/woocommerce/includes/libraries/wp-async-request.php(147): ActionScheduler_AsyncRequest_QueueRunner->handle()
#17 /home/m/mozhayev/rybolov59.rf/public_html/wp-includes/class-wp-hook.php(287): WP_Async_Request->maybe_handle('')
#18 /home/m/mozhayev/rybolov59.rf/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters('', Array)
#19 /home/m/mozhayev/rybolov59.rf/public_html/wp-includes/plugin.php(478): WP_Hook->do_action(Array)
#20 /home/m/mozhayev/rybolov59.rf/public_html/wp-admin/admin-ajax.php(190): do_action('wp_ajax_nopriv_...')
#21 {main}
  thrown in /home/m/mozhayev/rybolov59.rf/public_html/wp-content/plugins/wooms-extra/inc/OrderSender.php on line 519
И за фатальных ошибок 


Отправлено из мобильной Почты Mail.ru
